content/management/v1/core-resources/deployments/intro.md

```diff
@@ -6,3 +6,5 @@ The endpoint /v1/spaces/:space_id/deployments can be used to deploy branches.
 
 You can read more about branches [here](https://www.storyblok.com/docs/setup-branches-and-releases#pipeline-stage).
 
+It's important to note that stories created via the Management API would be created in the default branch. However, using additional parameters, the content can be published to specific branches (or pipeline stages, as they referred to them).
```

Comment:
Please replace it with:

Stories created via the Management API are created in the default branch (Preview). Subsequently, they can be released to other branches (e.g. Staging, Production) by creating a branch deployment.

Reply:
I'd like to suggest adding the information "according to the "source of sync" option". @manuelschroederdev @iamfortune  what do you think : 

Stories created via the Management API are created in the default branch (Preview). Subsequently, they can be released to other branches (e.g. Staging, Production **according to the "source of sync" option**) by creating a branch deployment.

Reply:
Yes, I like that. Can you please update the PR accordingly @iamfortune?

Reply:
Done 👍🏽